tests/test_version_range.py

```diff
@@ -209,9 +211,28 @@ def test_PypiVersionRange_raises_ivr_for_invalid_ranges(self):
         except InvalidVersionRange as ivre:
             assert str(ivre).startswith("Unsupported character")
 
+    def test_NpmVersionRange_from_native_with_compatible_with_version_operator(self):
+        npm_range = "^1.2.9"
+        expected = (
+            VersionConstraint(comparator=">=", version=SemverVersion(string="1.2.9")),
+            VersionConstraint(comparator="<", version=SemverVersion(string="2.0.0")),
+        )
+        version_range = NpmVersionRange.from_native(npm_range)
+        assert version_range.constraints == expected
```

Comment:
The function name says about testing `NpmVersionRange_from_native` but you're only testing the constraints.
How about:
- Modifying the function name accordingly or
- Changing the test to comply with the name

Reply:
Yeah, but that's what `from_native` is doing.. i.e returning  tuple of `VersionConstraint`.
And the  test `NpmVersionRange_from_native` only intendeds to test that specific `constraints` tuple.
Right?

Reply:
Yeah, That makes complete sense to me.
Will modify the tests to do assertion for `NpmVersionRange` instead of `constraints`

And I guess we have a similar issue with `GemVersionRange_from_native`, shall we do something about this too?

https://github.com/nexB/univers/blob/21864169341c4949bd40c931cd8450488de24d64/tests/test_version_range.py#L182-L189

Reply:
With tests, it's always the more the merrier :)